src/app/containers/RadioSchedule/Canonical/index.jsx

```diff
@@ -4,54 +4,81 @@ import { string } from 'prop-types';
 import styled from 'styled-components';
 import pathOr from 'ramda/src/pathOr';
 import {
+  GEL_GROUP_1_SCREEN_WIDTH_MAX,
+  GEL_GROUP_2_SCREEN_WIDTH_MIN,
   GEL_GROUP_2_SCREEN_WIDTH_MAX,
   GEL_GROUP_3_SCREEN_WIDTH_MIN,
-  GEL_GROUP_3_SCREEN_WIDTH_MAX,
   GEL_GROUP_4_SCREEN_WIDTH_MIN,
 } from '@bbc/gel-foundations/breakpoints';
 import {
+  GEL_SPACING,
   GEL_SPACING_DBL,
   GEL_SPACING_TRPL,
   GEL_SPACING_QUAD,
+  GEL_MARGIN_BELOW_400PX,
+  GEL_MARGIN_ABOVE_400PX,
 } from '@bbc/gel-foundations/spacings';
 import RadioSchedule from '@bbc/psammead-radio-schedule';
 import SectionLabel from '@bbc/psammead-section-label';
 import { Link } from '@bbc/psammead-story-promo';
+import { C_LUNAR } from '@bbc/psammead-styles/colours';
 import { ServiceContext } from '#contexts/ServiceContext';
 import processRadioSchedule from '../utilities/processRadioSchedule';
 import webLogger from '#lib/logger.web';
 
 const logger = webLogger();
 
+const RadioScheduleSection = styled.section.attrs(() => ({
+  role: 'region',
+  'aria-labelledby': 'Radio-Schedule',
+}))`
+  background-color: ${C_LUNAR};
+  padding: 0 ${GEL_MARGIN_BELOW_400PX};
+
+  /* To remove GEL Margins */
+  margin: 0 -${GEL_MARGIN_BELOW_400PX};
+  @media (max-width: ${GEL_GROUP_1_SCREEN_WIDTH_MAX}) {
+    padding-bottom: ${GEL_SPACING_DBL};
+  }
+
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
+    padding: 0 ${GEL_MARGIN_ABOVE_400PX};
+    margin: 0 -${GEL_MARGIN_ABOVE_400PX}; /* To remove GEL Margins */
+  }
+`;
+
 const MarginWrapper = styled.div`
-  @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
-    margin-top: ${GEL_SPACING_DBL};
+  margin: 0 auto;
+  padding-bottom: 0;
```

Comment:
Are all these paddings confirmed by UX?

Reply:
I used the spacings specified in the designs, however I made a bit of an error on the margins for the smaller breakpoints. Pushing those changes in a few

Reply:
Ok, I was actually gonna ask about the smaller breakpoints because I could just see it applied above 1008px.

I would suggest to apply the top margins to the `RadioScheduleSection` instead of the `RadioScheduleSectionLabel`.

Reply:
Same with the `margin-bottom: ${GEL_SPACING_TRPL};` in the `MarginWrapper`.

Reply:
I have made these changes.